### ISSUE TITLE
Python wrapper is ignoring seed

### ISSUE THREAD

**Issue by gatapia** [dmlc/xgboost#366]
Looking at the code it appears seed is being ignored in the python wrapper. Also in XGBModel please rename seed to random_state as this is more inline with sklearn semantics.

**Comment by tqchen** [comment]
python wrapper do take care of the seed, see https://github.com/dmlc/xgboost/blob/master/src/learner/learner-inl.hpp#L108.

For another part, please consider open a PR

**Comment by wwl89102** [comment]
I have tried for different seeds, but it seems workless for the rank:pairwise task. They all come out with the same results
